app/src/main/java/com/github/se/icebreakrr/ui/map/MapView.kt

```diff
@@ -349,3 +436,66 @@ fun MarkerOverlay(position: Offset, text: String) {
             )
       }
 }
+
+@Composable
+fun ProfileModal(
```

Comment:
`ProfileModal` in general:
Theres a lot of magic numbers, you know what to do 🥲

Reply:
No magic numbers allowed!